tests/automation/linked_device_group.spec.ts

```diff
@@ -1,100 +1,72 @@
-import { createGroup } from './setup/create_group';
-import { newUser } from './setup/new_user';
-import { sessionTestThreeWindows } from './setup/sessionTest';
+import { test_group_Alice_2W_Bob_1W_Charlie_1W } from './setup/sessionTest';
 import { leaveGroup } from './utilities/leave_group';
-import { linkedDevice } from './utilities/linked_device';
 import {
   clickOnTestIdWithText,
   waitForTestIdWithText,
 } from './utilities/utils';
 
-sessionTestThreeWindows(
+test_group_Alice_2W_Bob_1W_Charlie_1W(
   'Check group and name syncs',
-  async ([windowA, windowB, windowC]) => {
-    const [userA, userB, userC] = await Promise.all([
-      newUser(windowA, 'Alice'),
-      newUser(windowB, 'Bob'),
-      newUser(windowC, 'Charlie'),
-    ]);
-    const [windowD] = await linkedDevice(userA.recoveryPhrase);
-
-    const group = await createGroup(
-      'Testing group creation',
-      userA,
-      windowA,
-      userB,
-      windowB,
-      userC,
-      windowC,
-    );
+  async ({ aliceWindow2, groupCreated }) => {
     // Check group conversation is in conversation list on linked device
     await waitForTestIdWithText(
-      windowD,
+      aliceWindow2,
       'module-conversation__user__profile-name',
-      group.userName,
+      groupCreated.userName,
     );
   },
 );
 
-sessionTestThreeWindows(
+test_group_Alice_2W_Bob_1W_Charlie_1W(
   'Leaving group syncs',
-  async ([windowA, windowC, windowD]) => {
-    const [userA, userB, userC] = await Promise.all([
-      newUser(windowA, 'Alice'),
-      newUser(windowC, 'Bob'),
-      newUser(windowD, 'Charlie'),
-    ]);
-    const [windowB] = await linkedDevice(userA.recoveryPhrase);
-
-    const group = await createGroup(
-      'Testing leaving a group',
-      userA,
-      windowA,
-      userB,
-      windowC,
-      userC,
-      windowD,
-    );
+  async ({
+    aliceWindow1,
+    aliceWindow2,
+    bobWindow1,
+    charlie,
+    charlieWindow1,
+    groupCreated,
+  }) => {
     // Check group conversation is in conversation list of linked device
     await waitForTestIdWithText(
-      windowB,
+      aliceWindow2,
       'module-conversation__user__profile-name',
-      group.userName,
+      groupCreated.userName,
     );
     // User C to leave group
-    await leaveGroup(windowD, group);
+    await leaveGroup(charlieWindow1, groupCreated);
     // Check for user A for control message that userC left group
     // await sleepFor(1000);
```

Comment:
lot of commented out sleep code. Is it there for debugging purposes? Otherwise should be cleaned up.

Reply:
this is for Emily to choose if it can be removed or not, I have only done high level changes

Reply:
Cool, I see the linked device test is mentioned in the TODO.md so that is fine